configuration/package_manifests/virtual/nfpm.tedge-minimal.yaml

```diff
@@ -26,16 +26,19 @@ overrides:
   # slightly different formats
   apk:
     depends:
+        - mosquitto
```

Comment:
I didn't quite get why we want to add this additional dependency to the `minimal` package. It was supposed to be `minimal`, right? It looks weird that `tedge-minimal` pulls in an extra dependency that regular `tedge` doesn't have, as it's supposed to be minimal than `tedge` itself.

Reply:
This isn't an additional dependency as previously when`tedge-minimal` was being installed it would also install `mosquitto` (as `mosquitto` was a dependency of `tedge`). So in order to not break things for people already using the `tedge-minimal`, it `mosquitto` was added to the `tedge-minimal` dependencies.

Reply:
Though I must admit the `tedge-minimal` is not so practical in my opinion...so we might eventually phase it out.